docs/user-guides/registered.md

```diff
@@ -252,7 +258,7 @@ You must not spend the selected UTXO, because it needs to be consumed later in t
 
 ---
 
-Finally, the wizard outputs the command for obtaining signatures. You use this command in the next step (register-2 wizard). You must run it on a newly prepared machine with the main chain cold signing key.
+Finally, the wizard outputs the command for obtaining signatures. You use this command in the next step (register-2 wizard). You must run it on a newly prepared machine with the main chain cold signing key (e.g. `cold.skey`).
```

Comment:
Please use suggestive language instead of "must run it" and also highlight the offline nature of that machine, just as a suggestion. Eg.
"We suggest the command to be run on an offline machine, as to not expose the Cardano cold.skey to the internet, and return to the online machine to perform the register-2 wizard."
I'm a bit outdated here, are there still 3 commands, or you have reduced them to 2? Here I mean whichever step  needs the cold.skey to produce the signature.